site/docs/components/dropdown.mdx

```diff
@@ -131,6 +137,8 @@ Select dropdown serves in most use cases when the user needs to select one of 4
 
 Select multi-selection variant allows the user to select one or more options simultaneously. Note! Select dropdown does not feature filtering options by typing. When this feature is needed or there are a large number of options available, consider using consider using [Combobox](#combobox-multi-selection).
 
+An icon depicting the specific functionality of the multiselect variant in question should be added before the text input.  
```

Comment:
This note can be a bit misleading since the developer cannot control where the icon is added - it will be added to the same position always.

It would be more appropriate to say that these components support having an icon if needed.

Reply:
- [ ] Improve wording